py/selenium/webdriver/common/action_chains.py

```diff
@@ -17,6 +17,9 @@
 
 """The ActionChains implementation,"""
 import warnings
+from typing import List
```

Comment:
use `from typing import List, ...` format

Reply:
> 

Yes, I could do that, but when I run lynting, it is not happy. It is asking me to import in separate lines.

Reply:
that's a weird lint setting 🤷 
you could also simply do `import typing` and then use typing.List or similar in the code

Reply:
Yes, I can just do `import typing`, but everywhere I have to suffix `typing.List`, `typing.Union`, `typing.Optional`. So to avoid suffixing module name, I am import each type separately in import statements.

Reply:
> that's a weird lint setting 🤷
@mykola-mokhnach it would appear that isort default configuration is causing imports to be broken up to multi-lines
This can be fixed by either creating an `.isort.cfg` (or adding the switch in the tox.ini) and using `multi_line_output`.  There are a number of settings possible, and I'm not sure what selenium group dictates for your style. 
 https://pycqa.github.io/isort/docs/configuration/multi_line_output_modes.html